src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
+  const basePriceOfDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const TermDiscount = 20;
```

Comment:
The variable `TermDiscount` should be consistently named. Consider renaming it to `shortTermDiscount` to match the naming convention of `longTermDiscount`.